http/template/tag_packer.go

```diff
@@ -60,3 +63,41 @@ func TagPacker(env trails.Environment, filesys fs.FS) func(string, bool) html.HT
 		}
 	}
 }
+
+// AssetURI encloses the environment and filesystem so when called executing a template,
+// emits valid URI for client side static and bundled assets.
+//
+// TODO:
+//
+// - configurable OriginHost would allow for support of a CDN and custom Vite server origin
```

Comment:
The goal here was to move this into the direction of supporting a configurable Host for assets.  If we do that, then technically we shouldn't even need the `env.IsDevelopment()` case, as the host env would just be set to `http://localhost:8080/` in our local environments.  Having a configurable host means CDN support as well as customized origin with a dev server like `vite serve` - we don't need that with Tax going away, but it would have been nice to have when Tax existed.